contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -854,7 +875,7 @@ contract DualAggregator is OCR2Abstract, OwnerIsCreator, AggregatorV2V3Interface
    * @param roundId the aggregator round of the target report
    */
   function getAnswer(uint256 roundId) public view virtual override returns (int256) {
-    if (roundId > 0xFFFFFFFF) return 0;
+    if (roundId > latestRound()) return 0;
```

Comment:
i think we want to keep the previous behavior, and then add a new case where we `revert` with a `RoundNotFound` error when `(roundId > latestRound())`

Reply:
In the OCR2Aggregator, when we get an Answer that still not exist it doesn't revert, returns 0.

I didn't want to change the behaviour because maybe it would break the smart contracts of our users (clients) that already consume our feeds.

Reply:
okay, in that case i think we simply keep the existing logic, since all values in solidity are initialized (ie there is no `null`) `return s_transmissions[uint32(roundId)].answer;` will return `0` as expected. this check exists to make sure that the `uint256` doesn't go out of bounds for the mapping that uses `uint32` as it's key